api/anilist/src/main/kotlin/com/imashnake/animite/api/anilist/AnilistMediaRepository.kt

```diff
@@ -33,10 +34,14 @@ class AnilistMediaRepository @Inject constructor(
                 )
             )
             .executeCacheAndNetwork()
-            .asResult { it.page!! }
+            .asResult {
+                it.page!!.media.orEmpty().filterNotNull().map {
+                    cringe -> Media.Medium(cringe)
```

Comment:
Why do we have a name here and why is it on a separate line

Reply:
`it` is used by `asResult` so we need to specify it explicitly. As for the seperate line, it looks better.

Reply:
> it is used by asResult so we need to specify it explicitly

We don't need to, but it is nice to have

> As for the seperate line, it looks better.

No 😃 

cc @Iannnr on that one too

Name it something sensible

Reply:
Okay